packages/components/src/card/card-divider/hook.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../../divider').DividerProps, 'hr'>} props
+ */
+export function useCardDivider( props ) {
+	const { className, ...otherProps } = useContextSystem(
+		props,
+		'CardDivider'
+	);
+
+	const classes = useMemo(
+		() =>
+			cx(
+				styles.Divider,
+				styles.borderColor,
+				'components-card__divider',
```

Comment:
Should we make sure to add these classnames back to all of the other components as well?

Reply:
Yup! Done in https://github.com/WordPress/gutenberg/pull/32243/commits/6f0db934e5c740562909c65fd062160bd0c6bb53